### PR TITLE
Handle multiple animationName props

### PR DESCRIPTION
**Pull request by jaredbrookswhite**
Method to handle #675.

### CODE REVIEW

File: src/plugins/keyframes-plugin.js [hunk 1]
```diff
@@ -9,11 +9,30 @@ export default function keyframesPlugin(
...
       let value = style[key];
-      if (key === 'animationName' && value && value.__radiumKeyframes) {
-        const keyframesValue = (value: Keyframes);
-        const {animationName, css} = keyframesValue.__process(config.userAgent);
-        addCSS(css);
-        value = animationName;
+      if (
+        key === 'animationName' &&
+        value &&
+        (value.__radiumKeyframes || Array.isArray(value))
```
Comment (kylecesmat) [inline src/plugins/keyframes-plugin.js]: Could we do this check above the `if` block?

```
let value = style[key];
const isKeyframeArray = Array.isArray(value);

if (
  key === 'animationName' &&
  value &&
  (value.__radiumKeyframes || isKeyframeArray)
) {
  if (isKeyframeArray) {
   ...etc
  }
```

File: src/plugins/keyframes-plugin.js [hunk 2]
```diff
@@ -9,11 +9,30 @@ export default function keyframesPlugin(
...
+              return animationName;
+            })
+            .join(', ');
+        } else {
+          const keyframesValue = (value: Keyframes);
+          const {animationName, css} = keyframesValue.__process(
+            config.userAgent,
+          );
+          addCSS(css);
+          value = animationName;
```
Comment (kylecesmat) [inline src/plugins/keyframes-plugin.js]: can we reduce the duplicate logic here and extract this to a shared method?

ie:

```
const processKeyframeStyle = (value) => {
  const keyframesValue = (value: Keyframes);
  const { animationName, css } = keyframesValue.__process(config.userAgent);
  addCSS(css);
  return animationName;
}
```

```
if (Array.isArray(value)) {
  value = value.map(processKeyframeStyle).join(', ');
} else {
  value = processKeyframeStyle(value);
}
```

**Comment by kylecesmat** [comment]
I believe you need to rebase/merge the upstream `master` branch. Looks like there is a merge conflict with the remote `app.jsx`

**Comment by jaredbrookswhite** [comment]
updated and done. sorry for the confusion, not in my normal dev environment today.

**Comment by kylecesmat** [comment]
This has been released in `radium@0.25.0`

**Pull request merged**